pkg/types/alpine/v0.0.1/entry.go

```diff
@@ -322,16 +322,21 @@ func (v V001Entry) CreateFromArtifactProperties(ctx context.Context, props types
 
 	re.AlpineModel.PublicKey = &models.AlpineV001SchemaPublicKey{}
 	publicKeyBytes := props.PublicKeyBytes
-	if publicKeyBytes == nil {
-		publicKeyBytes, err = ioutil.ReadFile(filepath.Clean(props.PublicKeyPath.Path))
+	if len(publicKeyBytes) == 0 {
+		if len(props.PublicKeyPath) != 1 {
```

Comment:
nit: if you end up pushing up another patch, could we rename `PublicKeyPath` to `PublicKeyPaths`?

Reply:
done.